esmvaltool/recipes/recipe_flato13ipcc.yml

```diff
@@ -692,6 +722,70 @@ diagnostics:
         # Plot style
         styleset: CMIP5
 
+  # **********************************************************************
+  # Flato et al. (2013) - IPCC AR5, chap. 9
+  # similar to fig. 9.14
+  # **********************************************************************
+  # SST zonal mean and equatorial, multi model mean
+  # **********************************************************************
+
+  fig09-14:
+    description: IPCC AR5 Ch. 9, Fig. 9.14 (sst error, zonal mean and equatorial)
+    themes:
+      - phys
+    realms:
+      - ocean
+    variables:
+      tos_zm:
+        short_name: tos
+        preprocessor: zonal_mean
+        reference_dataset: HadISST
+        mip: Omon
+        field: T2Ms
+      tos_eq:
+        short_name: tos
+        preprocessor: equatorial
+        reference_dataset: HadISST
+        mip: Omon
+        field: T2Ms
+    additional_datasets:
+      - {dataset: HadISST, project: OBS, type: reanaly, version: 1, start_year: 1979, end_year: 1999, tier: 2}
```

Comment:
since all the start_year/end_year values appear to be the same, it might make more sense to define them in the variable

Reply:
Makes sense.